### DISCUSSION TITLE
UNECE

### DISCUSSION THREAD

**Discussion by paula-fel** [transport-data/community#8] [Sources]
Includes the following indicators:

- passenger activity
- freight activity
- vehicle stock
- infrastructure network

Geography:

- UNECE countries (Europe, North America, Central & Western Asia)

Relevant URLs:

- https://w3.unece.org/PXWeb/en

**Comment by blackburnstat** [comment]
Thanks!
Two comments:
1. In addition to vehicle stock (i.e. total fleet) we have new registrations.
2. Western Asia (in the UN context) ≈ Middle-East; we don't have data for those countries.
